### PR TITLE
Update French translations

### PR DESCRIPTION
**Pull request by Jean-BaptisteC**
This PR finish work started here #22646 
I have:
- Improves and add new French translations
- Fix the name of this [translations file](https://github.com/Jean-BaptisteC/dbeaver/blob/devel/plugins/org.jkiss.dbeaver.ui.charts/OSGI-INF/l10n/bundle_fr_properties)

### CODE REVIEW

**Comment by LonwoLonwo** [comment]
Hello @Jean-BaptisteC 

Thank you for your contribution! 
We will look at it.

File: plugins/org.jkiss.dbeaver.model/OSGI-INF/l10n/bundle_fr.properties [hunk 1]
```diff
@@ -75,3 +75,41 @@ meta.org.jkiss.dbeaver.model.navigator.DBNResource.resourcePath.name=Chemin
 meta.org.jkiss.dbeaver.model.navigator.DBNResource.resourceLocation.name=Emplacement
 meta.org.jkiss.dbeaver.model.navigator.DBNResource.resourceLastModified.name=Derni\u00E8re modification
 meta.org.jkiss.dbeaver.model.navigator.DBNResource.resourceSize.name=Taille
+extension-point.org.jkiss.dbeaver.serialize.name=S\u00e9rialiseurs d'objets
+DateFormatter.time.general.timezone.label=Fuseau horaire
+DateFormatter.time.general.timezone.description=Fuseau horaire. Si non d\u00e9fini, utilise le fuseau horaire de la valeur elle-m�me
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.model/OSGI-INF/l10n/bundle_fr.properties]: `m�me` - something happened in this file with encoding. Please take a look.

Comment (Jean-BaptisteC) [reply plugins/org.jkiss.dbeaver.model/OSGI-INF/l10n/bundle_fr.properties]: Nice catch

File: plugins/org.jkiss.dbeaver.ext.postgresql/OSGI-INF/l10n/bundle_fr.properties [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreSetting.value.name=Valeur
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreProcedure.objectId.name=ID de l'objet
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreProcedure.aggregate.name=Regroup\u00e9
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreProcedure.body.name=Corps
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreProcedure.execCost.name=Co\u00fbt d'ex\u00e9cution
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreProcedure.language.name=Langage
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreProcedure.owner.name=Propri\u00e9taire
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreProcedure.owner.description=Le propri\u00e9taire de la fonction
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreProcedure.returnType.name=Type retourn\u00e9
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreProcedure.strict.description=La fonction retourne NULL si l?un des arguments est NULL. Dans ce cas, la fonction ne sera pas appel\u00e9e du tout. Les fonctions qui ne sont pas "strictes" doivent \u00eatre pr\u00e9par\u00e9es pour accepter des valeurs NULL.
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.ext.postgresql/OSGI-INF/l10n/bundle_fr.properties]: It looks like `l?un` also has unexpected encoding.

File: plugins/org.jkiss.dbeaver.ext.postgresql/OSGI-INF/l10n/bundle_fr.properties [hunk 3]
```diff
@@ -0,0 +1,347 @@
...
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndex.primary.name=Primaire
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndex.primary.description=Cet index repr\u00e9sente la cl\u00e9 primaire de la table (dans ce cas, le champ unique doit \u00e9galement \u00eatre vrai)
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndex.ready.name=Pr\u00eat
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndex.ready.description=Si true, l?index est pr\u00eat \u00e0 ajouter des donn\u00e9es. Une valeur false indique que l?index est ignor\u00e9 pour INSERTS/UPDATES.
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndex.unique.description=Index unique
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndex.valid.name=Valide
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndex.valid.description=Si true, l?index peut \u00eatre utilis\u00e9 dans les requ\u00eates. False signifie que l?index est peut-\u00eatre incomplet : il sera cependant modifi\u00e9 par les commandes INSERT/UPDATE, mais il est s\u00fbr de l?utiliser dans des requ\u00eates impossibles. Si elle est unique, la propri\u00e9t\u00e9 d?unicit\u00e9 n?est pas non plus garantie.
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.ext.postgresql/OSGI-INF/l10n/bundle_fr.properties]: `l?index`, And I see more issues with the apostrophe symbol. Could you please fix it?

File: plugins/org.jkiss.dbeaver.ext.postgresql/OSGI-INF/l10n/bundle_fr.properties (hunk 2 above)
Comment (Jean-BaptisteC) [reply plugins/org.jkiss.dbeaver.ext.postgresql/OSGI-INF/l10n/bundle_fr.properties]: Fixed

File: plugins/org.jkiss.dbeaver.ext.postgresql/OSGI-INF/l10n/bundle_fr.properties (hunk 3 above)
Comment (Jean-BaptisteC) [reply plugins/org.jkiss.dbeaver.ext.postgresql/OSGI-INF/l10n/bundle_fr.properties]: Fixed

**Comment by Jean-BaptisteC** [comment]
Adjusted some translations
Fixed wrong encoding and question mark

**Review by LonwoLonwo — APPROVED** [review]

**Comment by LonwoLonwo** [comment]
For the QA Team - you can look at PostgreSQL metadata object fields at least (columns, procedures, indexes).

**Review by E1izabeth — APPROVED** [review]

**Comment by MashaKorax** [comment]
verified

**Pull request merged**